app/components/add-validations-to-form.js

```diff
@@ -8,12 +8,15 @@ import { getLocalFileContentAsText } from '../utils/get-local-file-content';
 import { FORM, RDF } from '../utils/rdflib';
 import { sym as RDFNode } from 'rdflib';
 import { areValidationsInGraphValidated } from '../utils/validation-shape-validators';
-
-const REGISTERED_FORM_TTL_CODE_KEY = 'validationFormTtlCode';
+import {
+  OBSERVER_KEY,
+  deregisterFromForkingStoreObservable,
+} from '../utils/deregister-forking-store-observable';
 
 export default class AddValidationsToFormComponent extends Component {
   @tracked builderStore;
   @tracked builderForm;
+  @tracked isComponentNotShown;
```

Comment:
Note: not sure if this was an oversight, or intentionally added to match the arg. but tracked properties are unrelated to the arguments so this wasn't needed.

Reply:
Thats good to know because it felt like this was always needed..